### ISSUE TITLE
Build error after update Ionic CLI to 4.10.1

### ISSUE THREAD

**Issue by hsusyh** [Microsoft/vscode-cordova#493]
## Actual Behavior
when running the debugger, it shows
```
ERROR] Unable to find command: /usr/local/bin/node /usr/local/bin/ionic cordova build ios
   _             _
  (_) ___  _ __ (_) ___
  | |/ _ \| '_ \| |/ __|
  | | (_) | | | | | (__
  |_|\___/|_| |_|_|\___|  CLI 4.10.1
  Usage:
    $ ionic <command> [<args>] [--help] [--verbose] [--quiet] [--no-interactive] [--no-color] [--confirm] [options]
  Global Commands:
    config <subcommand> ............. Manage CLI and project config values (subcommands: get, set, unset)
    docs ............................ Open the Ionic documentation website
    info ............................ Print project, system, and environment information
    init ............................ (beta) Initialize existing projects with Ionic
    login ........................... Login to Ionic Appflow
    logout .......................... Logout of Ionic Appflow
    signup .......................... Create an account for Ionic Appflow
    ssh <subcommand> ................ Commands for configuring SSH keys (subcommands: add, delete, generate, list, setup, use)
    start ........................... Create a new project
  Project Commands:
    build ........................... Build web assets and prepare your app for any platform targets
    capacitor <subcommand> .......... (beta) Capacitor functionality (subcommands: add, copy, open, run, sync, update) (alias: cap)
    cordova <subcommand> ............ Cordova functionality (subcommands: build, compile, emulate, platform, plugin, prepare, requirements, resources, run)
    deploy <subcommand> ............. Appflow Deploy functionality (subcommands: build)
    doctor <subcommand> ............. Commands for checking the health of your Ionic project (subcommands: check, list, treat)
    generate ........................ Automatically create framework features (alias: g)
    git <subcommand> ................ Commands relating to git (subcommands: remote)
    integrations <subcommand> ....... Manage various integrations in your app (subcommands: disable, enable, list)
    link ............................ Connect local apps to Ionic Appflow
    monitoring <subcommand> ......... Commands relating to Ionic Appflow Error Monitoring (subcommands: syncmaps)
    package <subcommand> ............ Appflow package functionality (subcommands: build)
    repair .......................... Remove and recreate dependencies and generated files
    serve ........................... Start a local dev server for app dev/testing (alias: s)
    ssl <subcommand> ................ (experimental) Commands for managing SSL keys & certificates (subcommands: generate)
```

## Expected Behavior
1. build the ionic app

## Software versions
- Ionic (Ionic CLI)  : 4.10.1
- Ionic Framework : ionic-angular 3.9.2
- @ionic/app-scripts : 3.2.0
- Cordova VS Code extension version: 1.7.0
- VSCode version: 1.30.2
- OS platform and version: macOS Mojave
- NodeJS version: V10.9.0
- Cordova version: 8.1.2
- Cordova-Android/iOS/Windows version: android 7.1.1, browser 5.0.4, ios 4.5.5

## Outputs (Include if relevant)
- Output of the Debug Console (View -> Toggle Debug Console):
```
Launching for ios (This may take a while)...
Launching app (This may take a while)...
[ERROR] Unable to find command: /usr/local/bin/node /usr/local/bin/ionic cordova build ios
```
- Output of the Cordova output channel (View -> Toggle Output -> Select Cordova in ListBox):
```
n/a
```
- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab):
```
n/a
```

**Comment by SounD120** [comment]
Hi @hsusyh . Just tested a fresh Ionic app created with `Ionic CLI 4.10.1` and it builds without errors on iOS. Please make sure that `Ionic CLI` and `Node.js` are installed correctly and are present in your `PATH` environment.
Here is my configuration:
```
Ionic:

   ionic (Ionic CLI)             : 4.10.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0
   @angular-devkit/build-angular : 0.12.4
   @angular-devkit/schematics    : 7.2.4
   @angular/cli                  : 7.2.4
   @ionic/angular-toolkit        : 1.3.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : not available
   Cordova Plugins       : not available

System:

   Android SDK Tools : 26.1.1 
   ios-deploy        : 1.9.4
   NodeJS            : v10.15.0 
   npm               : 6.4.1
   OS                : macOS Mojave
   Xcode             : Xcode 10.1 Build version 10B61
```

**Comment by hsusyh** [comment]
Hi @SounD120 this issue only happens when I try to build inside vscode-cordova tool, but when I tried to build in terminal it's fine. Here's my config:

```
Ionic:

   ionic (Ionic CLI)  : 4.10.1
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.1, browser 5.0.4, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic 5.2.8, cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.1, (and 15 other plugins)

System:

   ios-deploy : 2.0.0
   NodeJS     : v10.9.0 (/Users/cindytan/.nvm/versions/node/v10.9.0/bin/node)
   npm        : 6.2.0
   OS         : macOS Mojave
   Xcode      : Xcode 10.1 Build version 10B61
```

**Comment by ruslan-bikkinin** [comment]
Hi @hsusyh you are using nvm and node executable is located in your home directory, but there is still a link to `/usr/local/bin/node`. Try to run `which node` and see what output you'd get. If there are more than one link removes other ones that are not relate to nvm. Please let us know if it helps.

**Comment by hsusyh** [comment]
Hey @ruslan-bikkinin when I run `which node`, it returns only one link  `/Users/cindytan/.nvm/versions/node/v10.9.0/bin/node`

**Comment by ruslan-bikkinin** [comment]
@hsusyh well, you still need to find out why global node is pointing to `/usr/local/bin/node`. It could be a broken symlink, remains of previous node installations etc. Extension itself doesn't try to use some specific node executable locations. You can try to run `which node`, `which ionic` in VS Code terminal and compare it with the output produced in system terminal. Nevertheless, if you use nvm more likely that you shouldn't have `/usr/local/bin/node`.

**Comment by hsusyh** [comment]
@ruslan-bikkinin thanks I will take a look and get back to you

**Comment by hsusyh** [comment]
@ruslan-bikkinin so the node pointing to `/usr/local/bin/node` was a VS Code issue. But somehow my issue was fixed after the new release v1.8.0 of vscode-cordova. My node is still pointing to `/usr/local/bin/node` but I no longer have the error.

**Comment by ruslan-bikkinin** [comment]
@hsusyh I had similar issue and I could solve it by calling `nvm alias default v10.15.1`, where v10.15.1 is your nvm node version. The one only thing is that every time you are going to switch to another node version like `nvm use v8.0.0` you need to call this command again to make it work in VS Code. Anyway, I'm glad that your issue is resolved, so I'm closing it. Feel free to re-open if needed.
